lib/MusicBrainz/Server/Entity/URL/RateYourMusic.pm

```diff
@@ -1,11 +1,20 @@
-package MusicBrainz::Server::Entity::URL::Rateyourmusic;
+package MusicBrainz::Server::Entity::URL::RateYourMusic;
+use MusicBrainz::Server::Translation qw( l );
 
 use Moose;
 
 extends 'MusicBrainz::Server::Entity::URL';
 with 'MusicBrainz::Server::Entity::URL::Sidebar';
 
-sub sidebar_name { 'Rateyourmusic' }
+sub sidebar_name {
+    my $self = shift;
+
+    if ($self->url =~ m{^https?://(?:www.)?rateyourmusic.com/feature/}i) {
+        return l('Interview at Rate Your Music');
```

Comment:
`Rate Your Music` should not be localizable as they don’t have any localized name.
```suggestion
        return l(
            'Interview at {external_website_name}',
            { external_website_name => 'Rate Your Music' }
        );
```
As a bonus, this message can be reused with other websites in the future.
(We should probably have the same with streaming and others.)

Reply:
We discussed this on IRC but sadly while I liked the idea I realized it doesn't work for languages such as Estonian (probably neither for Finnish and other languages that modify word endings with cases) since those depend on the specific name of the site.